atom/browser/native_window_mac.mm

```diff
@@ -277,7 +278,29 @@ - (NSRect)window:(NSWindow*)window
 
 @end
 
-@interface AtomNSWindow : EventDispatchingWindow {
+@interface AtomPreviewItem : NSObject <QLPreviewItem>
+
+@property (nonatomic, retain) NSURL *previewItemURL;
+@property (nonatomic, retain) NSString *previewItemTitle;
+
+- (id)initWithURL:(NSURL *)anURL title:(NSString *)aTitle;
+
+@end
+
+@implementation AtomPreviewItem
+
+- (id)initWithURL:(NSURL *)url title:(NSString *)title {
```

Comment:
Similar comment here, looks like elsewhere in this file there isn't a space between the type and the `*`:

```
- (id)initWithURL:(NSURL*)url title:(NSString*)title {
```